addons/notes/src/addon.ts

```diff
@@ -16,7 +18,13 @@ export const withNotes = makeDecorator({
     const storyOptions = parameters || options;
 
     const { text, markdown, markdownOptions } =
-      typeof storyOptions === 'string' ? { text: storyOptions } : storyOptions;
+      typeof storyOptions === 'string'
+        ? {
+            text: storyOptions,
+            markdown: undefined,
```

Comment:
Is this mandatory?

Reply:
my IDE said, they are missing - normally it should not be necessary.
I guess there is a better way, e.g. 
```? {
  text: storyOptions
} as StoryOptions```

There may be an even better way, I'll check this later

Reply:
I did not found a class or interface of `StoryOptions`. Anyway, without specified options it will fail because it expect that values from that object. You can define it as default values, like:
```
const { text, markdown = undefined, markdownOptions = undefined } = { text: "" };
```
Here is an example in a [playground](http://www.typescriptlang.org/play/#src=declare%20var%20storyOptions%3A%20any%3B%0D%0A%0D%0A%0D%0Aconst%20%7B%20text%2C%20markdown%2C%20markdownOptions%20%3D%20undefined%20%7D%20%3D%0D%0A%20%20%20%20%20%20typeof%20storyOptions%20%3D%3D%3D%20'string'%0D%0A%20%20%20%20%20%20%20%20%3F%20%7B%0D%0A%20%20%20%20%20%20%20%20%20%20%20%20text%3A%20storyOptions%2C%0D%0A%20%20%20%20%20%20%20%20%20%20%20%20%7D%0D%0A%20%20%20%20%20%20%20%20%3A%20storyOptions%3B)